$OpenBSD$

Adjust the HTTP2 server call to the http2-5.4.0 API available on OpenBSD.

--- ../simplexmq-stable/src/Simplex/Messaging/Transport/HTTP2/Server.hs
+++ ../simplexmq-stable/src/Simplex/Messaging/Transport/HTTP2/Server.hs
@@ -85,8 +85,8 @@
 runHTTP2ServerWith_ expCfg_ clientFinished bufferSize setup http2Server = setup $ \(sniUsed, tls) -> do
   activeAt <- newTVarIO =<< getSystemTime
   tid_ <- mapM (forkIO . expireInactiveClient tls activeAt) expCfg_
   withHTTP2 bufferSize (run sniUsed tls activeAt) (clientFinished $ tlsUniq tls) tls `finally` mapM_ killThread tid_
   where
-    run sniUsed tls activeAt cfg = H.run cfg $ \req _aux sendResp -> do
+    run sniUsed tls activeAt cfg = H.run H.defaultServerConfig cfg $ \req _aux sendResp -> do
       getSystemTime >>= atomically . writeTVar activeAt
       http2Server sniUsed (tlsUniq tls) (tlsALPN tls) req (`sendResp` [])
